.github/workflows/conformance-e2e-v1.13.yaml

```diff
@@ -20,7 +20,7 @@ on:
   #   will disappear from the PR checks: please provide a direct link to the
   #   successful workflow run (can be found from Actions tab) in a comment.
   #
-  # pull_request: {}
+  pull_request: {}
```

Comment:
should this remain commented?

Reply:
nope this was just to get the tests to run here